README.md

```diff
@@ -1,2 +1,16 @@
 # rero-ils-tools
 RERO ILS misc tools for data manipulation.
+
+For local machines:
+    export PYTHONPATH=~/rero-ils-tools
+    export PATH=$PATH:~/rero-ils-tools/scripts/
+For PODs:
+    export PYTHONPATH=/invenio/src/rero-ils-tools
+    export PATH=$PATH:/invenio/src/rero-ils-tools/scripts/
+
```

Comment:
You can check the rendering on https://github.com/BadrAly/rero-ils-tools/tree/baa-item_type_modify

```suggestion
## Usage

### For local machines

1. Clone the repository.
1. Set the environment variables:

    export PYTHONPATH=~/rero-ils-tools
    export PATH=$PATH:~/rero-ils-tools/scripts/
    
### For PODs

    export PYTHONPATH=/invenio/src/rero-ils-tools
    export PATH=$PATH:/invenio/src/rero-ils-tools/scripts/
```